miden-crypto/src/dsa/rpo_falcon512/keys/mod.rs

```diff
@@ -4,7 +4,7 @@ use super::{
 };
 
 mod public_key;
-pub use public_key::{PubKeyPoly, PublicKey};
+pub use public_key::PublicKey;
```

Comment:
Do we no longer need to export the `PubKeyPoly` type? I thought we use it somehow in the VM.